src/imagelightbox.js

```diff
@@ -334,8 +334,8 @@
                 }
 
                 inProgress = true;
-                if (options.onLoadStart !== false) {
-                    options.onLoadStart();
+                if (onLoadStart !== false) {
+                    onLoadStart();
                 }
```

Comment:
The way it looks to me, if someone does set some function to the option "onLoadStart", it will conflict with the "arrows" and "caption" options.